docs/app/documentation/page/page.component.html

```diff
@@ -31,5 +31,23 @@ <h1 class="page-header h4">
   <div class="fixed-panel">
     <ngd-page-tabs [tabs]="currentItem?.tabs"></ngd-page-tabs>
     <ngd-page-toc [toc]="currentItem?.toc"></ngd-page-toc>
+
+    <ngd-hubspot-cta-component *ngIf="!isTabs" ctaId="14700578-ded0-4f32-b0bf-01b732af7855">
+      <img class="hs-cta-img" id="hs-cta-img-14700578-ded0-4f32-b0bf-01b732af7855"
```

Comment:
I would do a banner component instead of cta and encapsulate the image inside passing necessary parameters.

Or at least no inline styles please

Reply:
done